tests/core/usage_statistics/test_usage_statistics_data_context.py

```diff
@@ -106,8 +156,7 @@ def test_all_relevant_context_methods_emit_usage_stats(
 
         # Generate a set of dummy values to trigger the target method
         # and invoke the usage stats decorator without causing side-effects
-        signature = inspect.signature(method)
-        kwargs = {param: None for param in signature.parameters}
+        kwargs = {param: mock.MagicMock for param in required_args}
```

Comment:
Is there a spec we can use with this `MagicMock`?

Reply:
hmm I'm not sure - it's essentially an `Any`. We deemed this appropriate since we're just aiming to trigger usage stats (we don't care what happens in the actual function)